admin-dev/themes/new-theme/js/components/form/submittable-input.ts

```diff
@@ -24,90 +24,100 @@
  */
 
 import {showGrowl} from '@app/utils/growl';
+import ComponentsMap from '@components/components-map';
```

Comment:
We should definitly start to export multiple objects from these maps. so we're able to use single export